### PR TITLE
Adds more Kibana apps to shared attributes

### PR DESCRIPTION
**Pull request by lcawl**
This PR adds shared attributes for more of the Kibana apps and UIs.

### CODE REVIEW

File: shared/attributes.asciidoc [hunk 1]
```diff
@@ -123,13 +123,19 @@ ifdef::elasticsearch-root[]
...
 
-:uptime-app: Uptime app
-:logs-app: Logs app
-:metrics-app: Metrics app
-:siem-app: SIEM app
+:uptime-app:         Uptime app
+:logs-app:           Logs app
+:metrics-app:        Metrics app
+:siem-app:           SIEM app
+:ml-app:             *Machine Learning*
```
Comment (gtback) [inline shared/attributes.asciidoc:133]: It's possible I'm doing something wrong, but I'm not seeing the bold formatting coming through when using these attributes. Is that what you were trying to do?

File: shared/attributes.asciidoc [hunk 2]
```diff
@@ -123,13 +123,19 @@ ifdef::elasticsearch-root[]
...
 //////////
-Kibana app names
+Kibana app and UI names
 //////////
 
-:uptime-app: Uptime app
-:logs-app: Logs app
-:metrics-app: Metrics app
-:siem-app: SIEM app
+:uptime-app:         Uptime app
```
Comment (bmorelli25) [inline shared/attributes.asciidoc:129]:
```suggestion
:apm-app:             APM app
:uptime-app:         Uptime app
```

File: shared/attributes.asciidoc (hunk 1 above)
Comment (lcawl) [reply shared/attributes.asciidoc]: You're right, it doesn't work. I've removed the asterisks and can add them as necessary in the source files.

File: shared/attributes.asciidoc [hunk 3]
```diff
@@ -123,13 +123,21 @@ ifdef::elasticsearch-root[]
...
-:uptime-app: Uptime app
-:logs-app: Logs app
-:metrics-app: Metrics app
-:siem-app: SIEM app
+:apm-app:            APM app
+:uptime-app:         Uptime app
+:logs-app:           Logs app
+:metrics-app:        Metrics app
+:siem-app:           SIEM app
+:security-app:       Elastic Security app
```
Comment (lcawl) [inline shared/attributes.asciidoc:134]: FYI @benskelker I added this "security-app" attribute based on what I saw in https://github.com/elastic/stack-docs/pull/1317

Comment (benskelker) [reply shared/attributes.asciidoc:134]: Thanks @lcawl
@jmikell821 @DonNateR  - FYI

Comment (jmikell821) [reply shared/attributes.asciidoc:134]: @DonNateR can you add attributes for `Elastic Security` and `Elastic Agent` please?

**Pull request merged**